code.gs

```diff
@@ -550,7 +550,7 @@ function checkEvents (testDate) {
     body = [bodyPrefix, '\n']
         .concat(bodyBuilder)
         .concat(['\n\n ', bodySuffix1, '\n ', bodySuffix2, '\n'])
-        .concat('\n', isRunningOutdatedVersion() ? [bodySuffix3, ' ', _('here'), ':\n', baseGitHubProjectURL + 'releases/latest', '\n '] : [])
+        .concat('\n', isRunningOutdatedVersion() ? [bodySuffix3, '\n', baseGitHubProjectURL + 'releases/latest', '\n '] : [])
```

Comment:
It looks like you forgot to modify line 558 below (the htmlBody equivalent of this one). "Here" needs to be enclosed in a `<a></a>` tag in the HTML version and I am not really sure which way would it the best to do it (considering the problem you raised about word positioning in different languages as well). Maybe via the `string.format()` function?